editor/tasks/leiningen/pack.clj

```diff
@@ -29,7 +29,7 @@
                     "lib" ["libparticle_shared.dylib"]}
    "arm64-macos" {"bin" ["dmengine"]
                   "lib" ["libparticle_shared.dylib"]}
-   "x86-win32"     {"bin" ["dmengine.exe" "dmengine.pdb"]
```

Comment:
Symbols were added to show symbolicated callstack when we Build&Run, but editor doesn't support 32bit windows anymore. Which means we don't need this `pdb` in the editor
https://github.com/defold/defold/pull/5742

Reply:
We use unpack folder only for the editor: 
- build and run
- executables for building process and luajit
- libs e.g. particles etc

sp, everething we need to be able to run project or build it, not bundle. So, we can run anything on windows 32bit because support for win32 removed for a long time ago. we don't need this platform at all

Reply:
I think a bit a bit more and taking into account the comment above: we don't need `x86-win32` at all